src/composables/use-pages.ts

```diff
@@ -47,7 +47,7 @@ export default function usePages() {
   ]
 
   const route = useRoute()
-  const currentPageId = computed(() => route.value.name)
+  const currentPageId = computed(() => route.value?.name?.split('__')[0])
```

Comment:
Originally, this returned the i18n name like `about__en`, which would not match the ids in the `pages` array.